warehouse/jobs/runner.go

```diff
@@ -233,14 +205,16 @@ func (a *AsyncJobWh) startAsyncJobRunner(ctx context.Context) error {
 			_ = a.updateAsyncJobs(ctx, asyncJobStatusMap)
 			wg.Add(1)
 			go func() {
+				ticker := time.NewTicker(a.asyncJobTimeOut)
```

Comment:
Why a ticker? The following `select` is not inside a loop. Can't we get away with a timer that doesn't even need stopping?